test/functional/page_objects/header_page.js

```diff
@@ -162,10 +162,12 @@ export function HeaderPageProvider({ getService, getPageObjects }) {
       .findByLinkText(quickTime).click();
     }
 
-    async getToastMessage() {
-      remote.setFindTimeout(defaultFindTimeout);
-      return await remote.findDisplayedByCssSelector('kbn-truncated.toast-message.ng-isolate-scope')
-      .getVisibleText();
+    async getToastMessage(findTimeout = defaultFindTimeout) {
+      await remote.setFindTimeout(findTimeout);
```

Comment:
ah, so then instead of `remote.findDisplayedByCssSelector`, we'd add a new function to find so that we can stuff overriding the custom timeout in there, and then use `find.displayedByCssSelector` and pass the extra findTimeout?